odsApi_versioned_docs/version-7.3/technical-articles/upgrading-admin-and-security-to-pg16.md

```diff
@@ -0,0 +1,155 @@
+# Upgrading the Admin and Security Databases to PostgreSQL 16.x
+
+PostgreSQL 16.x is minimum supported version for Ed-Fi ODS / API v7.3. There are
+three available methods for updating instances of the Admin and Security
+database running on earlier versions of PostgreSQL to 16.x:
+
+* Restoring database backups to a PostgreSQL 16.x instance
+
+* Using the [`pg_upgrade`](https://www.postgresql.org/docs/16/pgupgrade.html)
+utility
+
+* Using logical replication
+
+This article provides a high-level overview of upgrading Admin and Security
+databases instances to PostgreSQL 16.x using database backups.
+
+:::info
+
+The details for performing a database upgrade are deployment-specific, and this
+ article describes the process when carried out on a single-server installation
+using version-specific directory names. Detailed information on performing
+version upgrades and additional upgrade methods are available in the official
+PostgreSQL documentation for [Upgrading a PostgreSQL
+Cluster](https://www.postgresql.org/docs/16/upgrading.html#UPGRADING).
+
+:::
+
+## Upgrading using backups
+
+### Step 1. Create backups of the Admin and Security databases
+
+Use the `pg_dump` utility from the currently installed version of PostgreSQL to
+create backups of each database.
+
+<details>
+<summary>Windows examples</summary>
+
+```none
+C:\Program Files\PostgreSQL\<old_version>\bin\pg_dump.exe -U pgadmin -d EdFi_Admin -f EdFi_Admin_backup.sql
+```
+
+```none
+C:\Program Files\PostgreSQL\<old_version>\bin\pg_dump.exe -U pgadmin -d EdFi_Security -f EdFi_Security_backup.sql
+```
```

Comment:
Recommendation: combine these two code blocks into one, and change the language from `none` to `powershell`.

Reply:
Thank you for those catches. I'll make the changes to address them.